orchestrator/services/processes.py

```diff
@@ -226,6 +228,18 @@ def _db_log_process_ex(pid: UUID, ex: Exception) -> None:
         raise
 
 
+def _get_process(pid: UUID) -> ProcessTable:
```

Comment:
If I'm not mistaken we probably already have a helper that will give you process details. Or is this needed due to the async stuff?

Reply:
Yes this helper function was in the api/endpoints/processes module, I needed it in services/processes so moved it here to prevent circular imports.